tests/unit/Directors.spec.ts

```diff
@@ -34,258 +23,456 @@ async function click (vm: any, id: string) {
+    const directorListUI = vm.$el.querySelectorAll('.director-list-item')
+    expect(directorListUI[0].querySelector('.cease-btn').disabled).toBe(false)
+
+    // check that Appoint New Director button is enabled
+    expect(vm.$el.querySelector('.new-director-btn').disabled).toBe(false)
   })
 
-  it('displays the edit Director form when enabled', async () => {
-    // Open the edit director
-    await vm.editDirectorName(0)
+  it('displays Appoint New Director form when button clicked', async () => {
+    // set enabled prop
+    wrapper.setProps({ componentEnabled: true })
+    await Vue.nextTick()
 
-    // Verify the correct data in the text input field
-    expect(vm.$el.querySelectorAll('.edit-director__first-name input')[0].value).toBe('Peter')
+    // confirm that flag is set correctly
+    expect(vm.componentEnabled).toEqual(true)
+
+    // check that Appoint New Director button is enabled
+    expect(vm.$el.querySelector('.new-director-btn').disabled).toBe(false)
+
+    // click Appoint New Director button
+    await click(vm, '.new-director-btn')
+
+    // check that button is hidden
+    expect(wrapper.find('.new-director-btn').isVisible()).toBe(false)
+
+    // check that form is showing
+    expect(wrapper.find('ul.new-director').isVisible()).toBe(true)
+
+    // check that inputs are showing
+    expect(wrapper.find('#new-director__first-name').isVisible()).toBe(true)
+    expect(wrapper.find('#new-director__middle-initial').isVisible()).toBe(true)
+    expect(wrapper.find('#new-director__last-name').isVisible()).toBe(true)
+  })
+
+  it('handles "ceased" action', async () => {
+    // click first director's cease button
+    await click(vm, '#director-1-cease-btn')
+
+    // check that button has changed to "undo"
+    expect(vm.$el.querySelector('#director-1-cease-btn').innerHTML).toContain('Undo')
+
+    // check that director is marked as ceased
+    expect(vm.$el.querySelector('#director-1 .director-status').innerHTML).toContain('Ceased')
+
+    // check that director object has the 'CEASED' action
+    expect(vm.allDirectors.filter(el => el.id === 1)[0].actions).toContain('ceased')
+  })
+
+  it('handles "undo ceased" action', async () => {
+    // click first director's cease button
+    await click(vm, '#director-1-cease-btn')
+
+    // click first director's undo cease button
+    await click(vm, '#director-1-cease-btn')
+
+    // check that button has changed back to "cease"
+    expect(vm.$el.querySelector('#director-1-cease-btn').innerHTML).toContain('Cease')
+
+    // check that director is not marked as ceased
+    expect(vm.$el.querySelector('#director-1 .director-status .v-chip')
+      .getAttribute('style')).toContain('display: none;')
+
+    // check that director object does not have the 'CEASED' action
+    expect(vm.allDirectors.filter(el => el.id === 1)[0].actions).not.toContain('ceased')
+  })
+})
+
+describe('Directors as a COOP (no sync)', () => {
```

Comment:
So this is the weirest thing. Some of the tests required `sync: false` and some didn't, so I broke up the describe block into 2.